.github/CODEOWNERS

```diff
@@ -1154,7 +1154,6 @@ x-pack/test_serverless/api_integration/test_suites/common/platform_security @ela
 
 
 # Data Discovery
-/x-pack/test/api_integration/services/data_view_api.ts @elastic/kibana-data-discovery
```

Comment:
I see this file was moved to [`x-pack/test/api_integration/services/data_view_api.ts`](https://github.com/elastic/kibana/blob/main/x-pack/platform/test/api_integration/services/data_view_api.ts) (in a separate PR). Why not add a CODEOWNER entry for the new location?